PrebidMobile/ConfigurationAndTargeting/Prebid.swift

```diff
@@ -132,6 +132,12 @@ public class Prebid: NSObject {
     /// If true, the sdk will add `includebidderkeys` flag inside the targeting object described in [PBS Documentation](https://docs.prebid.org/prebid-server/endpoints/openrtb2/pbs-endpoint-auction.html#targeting)
     public var includeBidderKeys = false
     
+    /**
+     * If true, the SDK will not check the PBS status during initialization. This will save initialization time
```

Comment:
Does it actually save time? Server status check is an async operation that just drops a console log message if the status endpoint doesn't respond with an OK status.

So if this change saves initialization time, the comment is ok. If it doesn't, it's better to remove the second sentence.